server/views/pages/createGoals/index.njk

```diff
@@ -133,14 +137,16 @@
               value: form.goals[(goalLoop.index0)].targetCompletionDate,
               fieldset: {
                 legend: {
-                  text: "When are they aiming to achieve this by?",
+                  text: "When are they aiming to achieve this goal by?",
                   classes: "govuk-fieldset__legend--s"
                 }
               },
               items: targetCompletionDateItems,
               errorMessage: errors | findError("goals[" + goalLoop.index0 + "].targetCompletionDate")
             }) }}
 
+            <h3 class="govuk-heading-s">Add steps to help them achieve this goal</h3>
```

Comment:
Again, another content change that is not written explicitly in the ticket, but instead is in the screenshot and we are expected to know to do it! 🙄